src/tribol/geom/CompGeom.cpp

```diff
@@ -227,7 +233,8 @@ TRIBOL_HOST_DEVICE FaceGeomException CommonPlanePair::checkFacePair( const MeshD
   m_face1_convex = IsConvex( x1_loc, y1_loc, mesh1.numberOfNodesPerElement() );
   m_face2_convex = IsConvex( x2_loc, y2_loc, mesh2.numberOfNodesPerElement() );
 
-  FaceGeomException interpen_err = this->computeOverlap3D(
+  // explicitly call compute overlap routine for common plane so CUDA can determine stack size
+  FaceGeomException interpen_err = CommonPlanePair::computeOverlap3D(
```

Comment:
Was this specifically the fix for the stack issue? Changing "this->" to "CommonPlanePair::" ?

I know mortar isn't ported, but for completeness and consistency should we make this change in the mortar and aligned mortar versions of checkFacePair etc.?